concepts/src/concept_csv_export.py

```diff
@@ -24,24 +24,47 @@
 DOCKER = False
 DOCKER_CONTAINER = "openmrs-sdk-mysql"
 VERSION = "2.3"
-LOCALES = ["en"]
-DEFAULT_LOCALE = "en"
+LOCALES = ["en","es"]
+DEFAULT_LOCALE = "es"
```

Comment:
Not related to this changeset, but note that this script only exports the FULLY_SPECIFIED and short names for the specified locales.

Reply:
Use the script arguments to set these.  These are weird defaults to set.

Reply:
ah, right, I changed these quickly, forgetting they were also arguments, will revert